### PR TITLE
#161058875 Make load more button only visible

### PR DESCRIPTION
**Pull request by llabake**
#### What Does This PR Do?
This PR ensures the load mote button is always fixed
#### Description Of Task To Be Completed
Make the load more button visible for only 9 or more events on the dashboard
Make the load more button position fixed
#### Any Background Context You Want To Provide?
N/A
#### How can this be manually tested?
* Login to the app
* Add events less than 9 the load more button should not be visible 
* Add as many events greater than 9 the load more button should be visible
* You should see the load more button without needing to scroll down the page
#### What are the relevant pivotal tracker stories?
[161058875](https://www.pivotaltracker.com/n/projects/2174978/stories/161058875)
#### Screenshot(s)
screen shot 2018-10-25 at 10 05 37 pm

### CODE REVIEW

**Comment by kenneth-kip** [comment]
image
The load more button is high up the page.
The load more button should go to the bottom of the page.
Just below the cards

File: client/pages/Event/EventsPage.jsx [hunk 1]
```diff
@@ -45,18 +46,55 @@ class EventsPage extends React.Component {
     });
   }
 
-  componentWillReceiveProps(nextProps) {
+  /**
+   *
```
Comment (kenneth-kip) [inline client/pages/Event/EventsPage.jsx]: Just curious why the two empty lines within the doc blocs

**Comment by kenneth-kip** [comment]
Works fine
image
change the height of `.event_footer` to 80%

File: client/pages/Event/EventsPage.jsx [hunk 2]
```diff
@@ -45,18 +46,55 @@ class EventsPage extends React.Component {
...
+    if (this.btnRef.current !== null) {
+      if (snapshot !== null) {
+        const btn = this.btnRef.current;
+        btn.scrollIntoView();
+      }
+    }
+  }
+
+  /**
+   *
```
Comment (kenneth-kip) [inline client/pages/Event/EventsPage.jsx]: Also here `https://github.com/AndelaOSP/Andela-Socials/pull/141/files#r230769859`
Mind adding some comment to document what this function does ?

File: client/reducers/initialState.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 export default {
   activeUser: {},
   event: {},
-  events: [],
+  events: {
+    events: [],
```
Comment (kenneth-kip) [inline client/reducers/initialState.js]: Do you mind changing this to something more appropriate ?
Like eventList or allEvents

**Comment by kenneth-kip** [comment]
Delete event is actually breaking.

**Comment by BolajiOlajide** [comment]
@llabake What's the status of this PR?

**Comment by BolajiOlajide** [comment]
@llabake What's the status of this PR?

**Pull request closed without merging**